### PR TITLE
[minor] add CORS middleware

### PR DESCRIPTION
**Pull request by Swaagie**
Prerequisite to allow browsers to request build details (dashboard etc)

### CODE REVIEW

File: lib/middlewares/index.js [hunk 1]
```diff
@@ -2,7 +2,8 @@
 
 var cookieParser = require('cookie-parser'),
     bodyParser = require('body-parser'),
-    debugParam = require('./debug');
+    debugParam = require('./debug'),
+    cors = require('cors');
```
Comment (3rd-Eden) [inline lib/middlewares/index.js]: https://www.npmjs.com/package/access-control

Comment (Swaagie) [inline lib/middlewares/index.js]: yeah we can do that as well :)

**Comment by 3rd-Eden** [comment]
+1

**Comment by Swaagie** [comment]
Released as `1.4.0`

**Pull request merged**